wildfly-elytron/pom.xml

```diff
@@ -519,21 +466,6 @@
     </dependencies>
 
     <profiles>
-        <profile>
-            <id>modularizedJdk</id>
-            <activation>
-                <jdk>[9,)</jdk>
-            </activation>
-            <properties>
-                <!-- [WFCORE-1431] remove SASL workaround -->
-                <modular.jdk.args>--add-modules java.sql --illegal-access=permit</modular.jdk.args>
-                <!-- use version of jboss-logging that works much better with JDK9 -->
-                <modular.jdk.props>-Djdk.attach.allowAttachSelf=true</modular.jdk.props>
-                <!-- 2.20.x doesn't start on JDK10-->
-                <version.surefire.plugin>2.19.1</version.surefire.plugin>
```

Comment:
@cam-rod I see that we re-added these options to the poms where they were removed (tests/base/pom.xml and pom.xml). Should we then re-add them here under the `<properties>` as well?

Reply:
Leaning to removing the `modular.jdk.props` argument entirely based on the other comments I added. The surefire version I think is set by the jboss-parent module by default, and shouldn't be an issue since JDK 11 is the minimum.